LayoutTests/fast/table/inline-table-margin-baseline-expected.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+.container {
+    border: solid 1px black;
+}
+.vertical {
+  writing-mode: vertical-rl
+}
+.block {
+    display:inline-block;
```

Comment:
Nit: `display: inline-block;` with a space, for consistency

Reply:
Fixed